tests/testthat/test-reporter-junit.R

```diff
@@ -0,0 +1,6 @@
+context("JUnitReporter")
+
+test_that("Junit reporter produces XML", {
+
```

Comment:
I thought mostly of tests since I know little about right and wrong when it comes to junit output.